src/main/resources/component-web.conf

```diff
@@ -351,7 +351,7 @@ mail {
         port = "25"
 
         # Protocol and encryption of the SMTP connection (SMTP, SMTPS or STARTTLS)
-        protocol = "smtp"
+        protocol = "SMTP"
```

Comment:
can't we rather/also make the reader smarter?

Reply:
enum values are case insensitive, so this wouldn't be unambiguously anymore